src/core/style/structure.scss

```diff
@@ -148,6 +148,18 @@ input {
   }
 }
 
+.visuallyhidden {
```

Comment:
This class is a common utility from HTML5 Boilerplate, so by not using the hyphen it will be easier to remember.